source/numerics.tex

```diff
@@ -1130,18 +1130,13 @@
 \begin{itemdescr}
 \pnum
 \notes
-the branch cuts are along the negative real axis.
+The branch cuts are along the negative real axis.
 
 \pnum
 \returns
-The complex natural (base e) logarithm of \tcode{x},
-in the range of a strip mathematically unbounded along the
-real axis and in the interval \crange{-i times pi}{i times pi}
-along the imaginary axis.
-When \tcode{x} is a negative real
-number,
-\tcode{imag(log(x))}
-is pi.
+The complex natural (base-$e$) logarithm of \tcode{x}. For all \tcode{x},
+\tcode{imag(log(x))} lies in the interval \crange{$-\pi$}{$\pi$}, and
+when \tcode{x} is a negative real number, \tcode{imag(log(x))} is $\pi$.
```

Comment:
Note that this is merely an editorial and typographic reformulation of the original wording. It is unclear how to reconcole this specification of `log` with Annex G of C11, which requires the phase the logarithm of a negative real number to depend on the sign of the imaginary part (which can be +/-0).